From 2d6b7abe1e6c65e1c5fa7215a3b367367e790827 Mon Sep 17 00:00:00 2001
From: Niko Tyni <ntyni@debian.org>
Date: Fri, 18 Nov 2016 15:22:04 +0200
Subject: Silence a 'used only once' warning in Test::Builder

Symptom of the problem:

 % perl -w -e 'use Test::Requires::Git'
 # git version 2.9.3
 Name "Test::Requires::Git::TODO" used only once: possible typo at /usr/share/perl/5.24/Test/Builder.pm line 2249.

Origin: backport, https://github.com/Test-More/test-more/commit/0f0c7d4d6c1b1953244b96ba4a367c27d85837b7#diff-e07e049db6ba650cd1605f4a5513b0ddR1342
Bug-Debian: https://bugs.debian.org/840968

Patch-Name: fixes/test-builder-warning.diff
---
 cpan/Test-Simple/lib/Test/Builder.pm | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cpan/Test-Simple/lib/Test/Builder.pm b/cpan/Test-Simple/lib/Test/Builder.pm
index a8e7bd95b..c1dba1c66 100644
--- a/cpan/Test-Simple/lib/Test/Builder.pm
+++ b/cpan/Test-Simple/lib/Test/Builder.pm
@@ -2246,6 +2246,7 @@ sub find_TODO {
     return unless $pack;
 
     no strict 'refs';    ## no critic
+    no warnings 'once';
     my $old_value = ${ $pack . '::TODO' };
     $set and ${ $pack . '::TODO' } = $new_value;
     return $old_value;
